python/lsst/sconsUtils/tests.py

```diff
@@ -127,18 +127,26 @@ def messages(self, test):
                     "false", "failed")
 
     def run(self, fileGlob):
+        """Create a test target for each file matching the supplied glob.
+        """
```

Comment:
The other methods in that file use a similar style so I'm not sure what the problem is.